### ISSUE TITLE
Proposal for a new fixed fee plan

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#1458]
Our current business model is to take 5% of all donations coming through the platform. We picked this because this is what most crowdfunding websites are doing. So it was easy to explain and justify to everyone when we started.

Today, we think that this business model might not be the best to achieve our goal of building the common infrastructure for open communities around the world (in the same way that Wordpress built the needed common infrastructure for the blogging revolution).

That's why we want to test a new tiered pricing model and get feedback from the community. Our initial plan would be to test this with European collectives (in euros). That way we don't impact the majority of our existing collectives and we can see how things are going in one region. It's above all a learning opportunity.

## Main objectives

- Reduce the friction for communities to create an open collective
- Grow the Open Collective platform and become financially sustainable

## Why a commission based business model might not be the best for all communities?

The first reaction from people when they hear that we take a commission on their income is [quite](https://twitter.com/fabpot/status/1057639979755683840?s=21) [negative](https://twitter.com/sebmck/status/887650323615145984) (5% platform fee and usually another 5% for the non profit fiscal host). It deters a lot of people from creating an open collective.

It hasn't been too much of a problem for open source communities because Open Collective is a real pain killer for them. Without Open Collective, there is really no other easy way to emit invoices to companies who want to support them financially. Each community would have to create and maintain a legal entity which is a massive overhead that could and should be mutualized (that's why we created [the Open Source Collective 501c6](https://opencollective.com/opensourcecollective) which is now already hosting 897 open source projects).

Also in the US, the concept of fiscal sponsorship is fairly well known. And the rate of 10% is a common industry rate. In Europe, much less so.

Once we get outside of online communities, open collective is not a pain killer (yet) but just a vitamin, an expensive one. In meat space, people can always accept cash. People trust one another much more as well. So they can receive a donation on a personal bank account on behalf of the community. All of that is a pain to manage but it's doable. Open Collective can definitely help but no way they are going to pay a 10% cut for this service.

That's why if we want to grow, we need to reduce the friction to get started.
It should be free to start a collective and get the first donations rolling.

We have also heard the feedback that people would much prefer pay a fixed amount every month rather than a commission so that they can better budget it and plan for it.

There are also some technical reasons why we may want to consider another business model: some payment methods don't allow us to automatically take a cut (e.g. paypal or manual wires to the bank account of the collective that is owned by the host). So it would be easier to eventually move everyone to a monthly bill for the platform fees. It will also make it easier for everyone's accounting. And backers won't feel like we are taking a cut out of their donation.

Last but not least, it would also change how we prioritize features. Instead of solely focusing on the amount of money going through the platform, we will focus more on other features that are needed by collectives to thrive (see also our blog post: [rethinking the tools for collectives](https://medium.com/open-collective/rethinking-the-tools-for-collectives-7ee540515c75)).

## New tiered pricing (proposal)

Here is a first proposal for a new pricing model. The monthly budget represents the amount of money a collective receives per month (regardless of the payment method used).
Today's revenue is based on last month (October 2018) with our existing 5% commission (see also below the Data section for the datasets).

monthly budget | price | number of collectives | expected revenue | today's revenue
------------------------- | ----- | --------------------- | ------- | ---------------
$0-$100 | free | 261 | 0 | $245/m
$101-$200 | $10/m | 60 | $600/m | $378/m
$201-$500 | $20/m | 39 | $780/m | $588/m
$501-$1,500 | $50/m | 27 | $1,300/m | $1,252/m
$1,501-$5,000 | $150/m | 20 | $3,000/m | $1,645/m
$5,001-$10,000 | $300/m | 2 (material-ui, wwcodeatl)| $600/m | $495/m
$10,001+ | $1,000/m | 3 (webpack, babel, sustainoss) | $3,000/m | $2,084/m
**TOTAL** | | 414 | **$9,280**/m | $6,687/m
Spending | | |  ~$60k/m
Loss | | | ~$50k/m
Run way | | | 20 months

Bear in mind that this is based on our current dataset. Of the 544 collectives that have been active in the past 3 months, 462 were in USD (84%), 61 in EUR (11%). 355 (65%) were open source communities hosted by the open source collective. As an initial test phase, such new pricing model would only be applied to collectives in EUR. So the "expected revenue" is really just a comparison point if we were to apply the same pricing to existing collectives.

On top of that, the collective will have to add the host fee if they decide to be hosted by another legal entity.
But given that outside of open source communities, it's more common to either already have a legal entity, or to trust someone in meat space to hold the money on their personal bank account on behalf of the community, most of them won't need to pay such host fee.

(As a side note, we should probably enable hosts to also charge a monthly or yearly fee instead of a commission to each of the collectives that they host.)

## Data

I made an export of all transactions of the past 3 months with the associated fees (13,027 transactions, [query](https://github.com/opencollective/opencollective/wiki/Queries-for-public-data-exports#transactions)). Everything is available in our public google drive on https://drive.opencollective.com ([direct link to data exports](https://drive.google.com/open?id=1EayoSNwyh5f_oNNjfdg5ulq5PnCvhvu1)).

Feel free to have some fun with those datasets. Just please share what you find with the community. It's always great to have fresh eyes looking at our data and bringing new perspectives.

#### Women Who Code

In October, WWCode had 17 active collectives that collected a total of $20k ($770 directly through us, the rest by manually adding external funds to the budget of the collectives). They only gave us $140 of platform fees. With this new pricing, they would have to contribute $1,110/m in platform fees (note: this would come from the budget of the 17 active collectives, not from the budget of the host). 

Note: if we decide to only apply this new business model to Europe, it wouldn't impact Women Who Code.

#### BrusselsTogether

In October, BrusselsTogether had 15 active collective that collected a total of €13k (€2k directly through us). They paid €100 of platform fees. With the new pricing, they would have to contribute €730/m in platform fees.

#### Open Collective Paris

4 collective active, total donations: €222, platform fees: €9. New pricing: €10.

#### Open Collective Europe

In October, 13 active collective, €1,450, €66 platform fees. With new pricing: €40/m.

## Other scenarios

I also looked at making the backers pay directly instead of the collectives.

Last month, we had 3k backers for a total of $137k donated (only looking at USD).
- 368 were organizations. They gave $93k (avg: $255, median: $25, total platform fees: $4.6k)
  - 300 of them gave between $100 and $1k (50 of them to more than one collective)
  - 25 gave more than $1k.
- 2,669 individuals gave a total of $43k (avg: $16, median: $5, total platform fees: $1.5k)

I think it would make sense to introduce a Pro plan for backers, especially geared towards companies. This would give them better reporting, consolidated invoices, purchase orders. All of that is extra work compared to individual backers and of great value to them.

E.g. $1k or $5k/year depending on company size
It will also incentivize them to spend more to amortize their investment (if they donate $10k/year, that's a 10% overhead, but if they give $100k, that's only a 1% overhead).
Based on the current numbers, I'd expect 25 companies to subscribe to a $1k/year pro plan and 10 of them to get on a $10k/year. That would make another $125k/year or $10k/month and would help us get to "ramen profitability".

## Comparables

To put things in perspective, those are some of the tools that collectives pay for:

- Meetup: $5/m for up to 50 members, $8/m unlimited, see https://www.meetup.com/pricing/
- Mailchimp: free for first 2k subscribers with mailchimp banner, $10/m up to 500 subscribers, $20 up to 1.5k, ... see https://mailchimp.com/pricing/
- NationBuilder: $35/m up to 5k members (1k emailable), $83 up to 15k members (3k emailable), see https://nationbuilder.com/pricing
- Wix: $8.50/m, $12.5/m with forms, $16.5 with store, see https://www.wix.com/upgrade/website
- Heroku: $7/m per dyno https://www.heroku.com/pricing
- Patreon: 5% commission + payment processor fees, see https://support.patreon.com/hc/en-us/articles/204606125-How-do-you-calculate-fees-

## Conclusion

I like the idea of a free tier with up to $100/month of donations and a cheap tier at $10/month that will get you up to $500/month of donations. For those amounts of donation, people don't need to worry about having a host (and paying additional host fees). This category represents today 75% of active collectives. And I think this category could grow a lot more. Especially in Europe.

I also like the idea of a pro plan for companies on open collective. It will incentivize them to donate more and to rally more communities that they want to support to the platform. We might want to do some research to see what would be a good price point for them. My intuition would be to offer a small organization plan at $1k/year, medium at $5k/y and large at $10k/y. What do you think? What would you love to get for this?

Please share your comments and ideas. The goal is to make sure we can sustain open collective as a platform (you still want us to be around for decades to come right?) and that we can enable more communities around the world to become sustainable while embracing the ethos of open source that are transparency, inclusiveness, collaboration.

## Next steps

- Gather feedback from the community
- Implement such new pricing for new collectives hosted by our own Open Collective Europe non profit (https://opencollective.com/europe).
- Gather more feedback, share the learnings

In parallel, if the idea of a pro plan for organizations on Open Collective is getting some interest, that's something that we could pursue in parallel and that would apply to all organizations on Open Collective, not just in Europe.

**Comment by alanna** [comment]
We need to reopen this discussion. 

I am about to send out a marketing campaign to fiscal sponsor orgs, and I do a lot in the space of host onboarding generally. To me, the fiscal hosting aspect is the most exciting part of Open Collective, and each fiscal host has a potential magnifying impact of all their Collectives combined. But it's not really helping us thrive if we don't get any fees from them. And many will not come on board at all if we stick to the 5% platform fee for large amounts added manually (which is too high on a $50,000 grant for example).

Currently, our documentation says we charge a 5% platform fee for everything (implied that includes manual payments). In practice, this is not happening because we have not built any way to actually collect or invoice for fees that are not automatically deducted from credit card transactions. I have felt uneasy because I can't really give clear answers about our fees to potential hosts. Technically they can do manual payments for free, but I don't want to emphasise that because I think they should be paying *something*.

I like the overall thinking of @xdamman's pricing table above, but I am also playing with other ideas. (Note this is for platform fees only - host fees are a whole other discussion.)

### idea: flat 2% fee on all funds added manually 

* calculate what they owe each month and charge their credit card on file
* similar to how other apps charge based on usage (like Mailchimp or Slack)
* could disincentivise transparency because they might not add funds to avoid charge
* their motivation add funds would be for accurate budget tracking and accounting
* removes weird incentives to avoid tipping into the next price "bracket" because it's a smooth curve instead of stair steps
* no need to change the 5% fee on credit card donations we have in place now

I don't think my idea is necessarily better that the model @xdamman proposed. Either one would be waaaay better than the situation we have currently.

> I like the idea of a free tier with up to $100/month of donations and a cheap tier at $10/month that will get you up to $500/month of donations. For those amounts of donation, people don't need to worry about having a host (and paying additional host fees). This category represents today 75% of active collectives. And I think this category could grow a lot more. 

I like the free tier idea too (but we can't call it tiers! that word is taken in our world already - call it price plan or something). I don't understand the part about not needing a fiscal host though. You mean Open Collective Inc will serve as host for all these small Collectives?

> I also like the idea of a pro plan for companies on open collective. It will incentivize them to donate more and to rally more communities that they want to support to the platform.

I think we should keep this discussion carefully seperate from anything related to a "pro" offering for sponsors or contributors who submit a lot of expenses. To me this issue is about how we charge for money flowing through fiscal hosts that's added manually and therefore we're not charging for. 

In conclusion: I think we have a very valuable tool to offer fiscal host organizations, but lack of a way to charge them for the biggest chunks of money in the system is hurting us, plus lack of clarity about pricing is blocking new ones from coming on board.

**Comment by RussKie** [comment]
> (5% platform fee and usually another 5% for the non profit fiscal host). It deters a lot of people from creating an open collective.

Looking at our collective, we're getting skimmed off 20% which is quite substantial
![image](https://user-images.githubusercontent.com/4403806/60483069-46214d00-9cd7-11e9-9584-adc284981b1d.png)
